### ISSUE TITLE
Lintian whinges about syntax in main.profile

### ISSUE THREAD

**Issue by jmb202** [pexip/os-lintian-pexip#1]
Lintian now says:
```
Please use spaces as separators in field Disable-Tags instead of commas in profile /usr/share/lintian/profiles/pexip/main.profile
```

We should probably take the hint.

**Comment by steve-pexip** [comment]
Fixed in 81490371bfc9249bc8bc851450ede5bb71b30953, adding to an os release now

**Comment by steve-pexip** [comment]
Merged in https://github.com/pexip/mcu/commit/3b437d42c7fdcc32f8809a391af50434b8d5300b
